VIDEO_5_SCRIPT.md

```diff
@@ -0,0 +1,67 @@
+
+## What is open governance
+
+Open governance is a "model of governance that promotes the principles of transparency, integrity, accountability, and stakeholder participation in support of democracy and inclusive growth. Driven by the participation of all the people that make up a project, company, or institution, it promotes engagement and democracy".
+
+In the same way, this model facilitates working with like-minded people living thousands of miles away to pursue and achieve a common goal. The intention is to break down individualism and hierarchical archetypes while upholding a system that gives autonomy and decision-making power to its members.
+
+Therefore, all relevant project data are public. The developed tasks are accessible and open to the participation of the community. This is intended to cultivate an environment of trust and respect for the user that encourages involvement and collaboration. In this atmosphere, adaptable and flexible, nothing is imposed or taken for granted.
+
+## Why AsyncAPI is committed to an Open Governance model
+
+Partnering with Postman was a crucial moment for assuring the efforts of the AsyncAPI Initiative. This event was the propellant to accelerate the transfer of AsyncAPI to an open government foundation to assure the community that a single company does not control the AsyncAPI Initiative. In doing so, the specification safeguarded against private interests.
+
+In the AsyncAPI governance model, you can find two essential roles: contributor and committer. A contributor is a person that contributes to the project code, docs, or other artifacts. A committer is a person that contributes regularly and is invited by other committers to manage the repository, to have more privileges, and to approve pull requests.
+
+To supervise the initiative, it was decided to create a Technical Steering Committee (TSC), made up of the people who contribute most actively to the project. Anyone with interest can become a member of the TSC. All you have to do is become a maintainer of one of the AsyncAPI projects. To become a maintainer, it is necessary to regularly contribute to one of the projects; after that, other maintainers will invite them to join. 
+
+The TSC's system of operation is by voting. An issue or debate is posted, and the members vote asynchronously and publicly on their decision. The value of the vote of all contributors is equal, thus ensuring the project that there is no risk of one company having a majority of the voters. That is why there is a rule that only 1/4 of the voters can be affiliated with the same company. Or 1/3 in case more maintainers need to be sought to bring the balance back to the force.
```

Comment:
```suggestion
The TSC's system of operation is by voting. An issue or debate is posted, and the members vote asynchronously and publicly on their decision. The value of the vote of all contributors is equal, thus ensuring the project that there is no risk of one company having a majority of the voters. That is why there is a rule that only 1/4 of the voters can be affiliated with the same company.
```

it is gov model implementation detail, not worth mentioning, someone can check it out in the charter if they want to know more, most important is 1/4 rule